lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -72,9 +72,9 @@ def colorized_formatted_backtrace(colorizer=::RSpec::Core::Formatters::ConsoleCo
           end
         end
 
-        def fully_formatted(failure_number, colorizer=::RSpec::Core::Formatters::ConsoleCodes)
+        def fully_formatted(failure_number, colorizer=::RSpec::Core::Formatters::ConsoleCodes, skip=0)
           lines = fully_formatted_lines(failure_number, colorizer)
-          lines.join("\n") << "\n"
+          lines.drop(skip).join("\n") << "\n"
```

Comment:
From what I remember, the first line of "lines" is the full example name and the second is a blank line. In html formatter the example name is already there when the exceptions are formatted, then it would be duplicated in the final output. That's why we drop the first 2 lines.